postprocessing/comstockpostproc/comstock.py

```diff
@@ -727,10 +727,11 @@ def add_geospatial_columns(self):
 
         # Show the dataset size
         logger.debug(f'Memory after add_geospatial_columns: {self.data.estimated_size()}')
+    
+    def _read_csv(self, path, columns, dtypes): return pl.read_csv(path, columns=columns, dtypes=dtypes)
```

Comment:
Need to abstract the function out since if I directly hijack/mock the pl.read_csv function, it need to be mocked all the pl.read_csv in the comstock.__init__() and that's not what we want.